### PR TITLE
Card Component Done.

### PR DESCRIPTION
**Pull request by ashifkhn**
Card Component is added in the component Library.

Please help me with the review.

Here's the link: https://deploy-preview-8--whyui.netlify.app/

### CODE REVIEW

**Review by ShoaibTamboli — COMMENTED** [review]
Card looks good to me , just beautify both HTML and CSS files

**Comment by ashifkhn** [comment]
> Card looks good to me , just beautify both HTML and CSS files

thanks. on it.

File: components/Card/styleCard.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+		cursor: pointer;
+		background-color: var(--colorGrey);
+		color: black;
+	}
+
+.imageWithShadow{
+	box-shadow: 10px 10px 8px #888888;
+}
+
+.boxShadow{
```
Comment (sakshi006) [inline components/Card/styleCard.css:97]: try using css variables

File: components/Card/styleCard.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+	.imageWithBadge:hover{
+		cursor: pointer;
+		background-color: var(--colorGrey);
+		color: black;
+	}
+
+.imageWithShadow{
+	box-shadow: 10px 10px 8px #888888;
```
Comment (sakshi006) [inline components/Card/styleCard.css:94]: try using css variables for color

**Review by sakshi006 — COMMENTED** [review]
Minor change requested, Other than that Looks good to me.

File: components/Card/styleCard.css (hunk 2 above)
Comment (ashifkhn) [reply components/Card/styleCard.css:94]: sure.

**Pull request merged**